app/components/developer_query_component.rb

```diff
@@ -18,6 +18,15 @@ def time_zones
       .map { |offset| [offset, "#{offset} #{t("developer_query_component.gmt")}"] }
   end
 
+  def role_types
+    %i[part_time_contract full_time_contract full_time_employment]
+      .map { |role| [role, RoleType.human_attribute_name(role)] }
+  end
+
+  def role_selected?(role_pair)
```

Comment:
If we are calling this `#role_selected?` we should change the name of line 10 to `#time_zone_selected?`.

Reply:
sure will do that.